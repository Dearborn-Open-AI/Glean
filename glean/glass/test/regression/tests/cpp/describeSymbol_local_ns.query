action: describeSymbol
args: "test/cpp//foo/fooFn/fooI/.decl"
