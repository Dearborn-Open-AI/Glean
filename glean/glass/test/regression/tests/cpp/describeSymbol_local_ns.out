[
    "@generated",
    {
        "comments": [],
        "kind": 15,
        "language": 1,
        "location": {
            "filepath": "test.cpp",
            "range": {
                "columnBegin": 11,
                "columnEnd": 19,
                "lineBegin": 34,
                "lineEnd": 34
            },
            "repository": "test"
        },
        "name": {
            "container": "foo::fooFn",
            "localName": "fooI"
        },
        "repo_hash": "testhash",
        "signature": "int fooI",
        "sym": "test/cpp//foo/fooFn/fooI/.decl",
        "sym_location": {
            "filepath": "test.cpp",
            "range": {
                "columnBegin": 11,
                "columnEnd": 19,
                "lineBegin": 34,
                "lineEnd": 34
            },
            "repository": "test"
        },
        "sym_other_locations": []
    }
]